### ISSUE TITLE
Exception on `cg --list`

### ISSUE THREAD

**Issue by abolotnov** [Mermade/openapi-codegen#80]
Besides installation, is there something else that I need to do to list templates?

```
kitty@catbox:~$ cg --list og
internal/modules/cjs/loader.js:583
    throw err;
    ^

Error: Cannot find module '/home/kitty/configs/nodejs.json'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (/usr/lib/node_modules/openapi-codegen/cg.js:71:48)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
```

**Comment by MikeRalphson** [comment]
Just checking, did you install globally? Will try and reproduce.

**Comment by MikeRalphson** [comment]
Should be fixed in v2.0.1
